build.psm1

```diff
@@ -347,7 +347,9 @@ function Start-PSBuild {
         Write-Warning @"
```

Comment:
@iSazonov, your last commit had 14 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Get-Content.Check Get-Content containing multi-byte chars.Should return last three lines two lines at a time for -ReadCount 2 and -Tail 3
```powershell
Expected $null or empty, but got @(@{InputObject=System.Object[]; SideIndicator==>}, @{InputObject=System.Object[]; SideIndicator==>}, @{InputObject=System.Object[]; SideIndicator=<=}, @{InputObject=System.Object[]; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Get-Content.Tests.ps1: line 341
341:             Compare-Object -ReferenceObject $expected -DifferenceObject $result | Should -BeNullOrEmpty
```
Get-Content.Check Get-Content containing multi-byte chars.Should return first three lines two lines at a time for -TotalCount 3 and -ReadCount 2
```powershell
Expected $null or empty, but got @(@{InputObject=System.Object[]; SideIndicator==>}, @{InputObject=System.Object[]; SideIndicator==>}, @{InputObject=System.Object[]; SideIndicator=<=}, @{InputObject=System.Object[]; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Get-Content.Tests.ps1: line 353
353:             Compare-Object -ReferenceObject $expected -DifferenceObject $result | Should -BeNullOrEmpty
```
Get-Service cmdlet tests.Get-Service filtering works for '{ Get-Service -DisplayName Net* }'
```powershell
Expected $null or empty, but got 
InputObject       SideIndicator
-----------       -------------
NcaSvc            =>
NcbService        =>
Netlogon          =>
Netman            =>
NetMsmqActivator  =>
NetPipeActivator  =>
netprofm          =>
NetSetupSvc       =>
NetTcpActivator   =>
NetTcpPortSharing =>
NlaSvc            =>
nsi               =>
NcaSvc            <=
NcbService        <=
Netlogon          <=
Netman            <=
NetMsmqActivator  <=
NetPipeActivator  <=
netprofm          <=
NetSetupSvc       <=
NetTcpActivator   <=
NetTcpPortSharing <=
NlaSvc            <=
nsi               <=

.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Get-Service.Tests.ps1: line 69
69:       Compare-object $services $servicesCheck | Out-String | Should -BeNullOrEmpty
```
Get-Service cmdlet tests.Get-Service filtering works for '{ Get-Service -Include Net* -Exclude *logon }'
```powershell
Expected $null or empty, but got 
InputObject       SideIndicator
-----------       -------------
Netman            =>
NetMsmqActivator  =>
NetPipeActivator  =>
netprofm          =>
NetSetupSvc       =>
NetTcpActivator   =>
NetTcpPortSharing =>
Netman            <=
NetMsmqActivator  <=
NetPipeActivator  <=
netprofm          <=
NetSetupSvc       <=
NetTcpActivator   <=
NetTcpPortSharing <=

.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Get-Service.Tests.ps1: line 69
69:       Compare-object $services $servicesCheck | Out-String | Should -BeNullOrEmpty
```
Get-Service cmdlet tests.Get-Service filtering works for '{ Get-Service -Name Net* | Get-Service }'
```powershell
Expected $null or empty, but got 
InputObject       SideIndicator
-----------       -------------
Netlogon          =>
Netman            =>
NetMsmqActivator  =>
NetPipeActivator  =>
netprofm          =>
NetSetupSvc       =>
NetTcpActivator   =>
NetTcpPortSharing =>
Netlogon          <=
Netman            <=
NetMsmqActivator  <=
NetPipeActivator  <=
netprofm          <=
NetSetupSvc       <=
NetTcpActivator   <=
NetTcpPortSharing <=

.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Get-Service.Tests.ps1: line 69
69:       Compare-object $services $servicesCheck | Out-String | Should -BeNullOrEmpty
```